server/src/main/java/com/tencent/rss/server/ShuffleTaskManager.java

```diff
@@ -273,9 +274,9 @@ public long requireBuffer(int requireSize) {
   public byte[] getFinishedBlockIds(
       String appId, Integer shuffleId, Integer partitionId) throws IOException {
     refreshAppId(appId);
-    if (useMultiStorage) {
-      multiStorageManager.prepareStartRead(appId, shuffleId, partitionId);
-    }
+    Storage storage = storageManager.selectStorage(new ShuffleDataReadEvent(appId, shuffleId, partitionId));
```

Comment:
it's better to add comment to clarify what metrics will be updated

Reply:
OK